Bash_scripts/HW20/roles/hw202/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+---
+#Get info from files
+- name: "Get variables from file"
+  import_tasks: template.yml
+
+#Copy to Ubuntu
+- name: "Copy files to hosts"
```

Comment:
Use variables and make only one task for both OSs

Reply:
done